docs/operator-guide.md

```diff
@@ -239,6 +239,17 @@ downloaded and validated every 30 seconds. The result of the test is published a
 the test fails, a detailed error message is logged in stderr. If the setup phase fails, an error message is logged as
 well and the program immediately exits with non-zero status.
 
+### Trivy Proxy configuration options
+
+These options are only useful when the trivy proxy is deployed but the Keppel API and janitor
+
+| Variable | Default | Explanation |
+| -------- | ------- | ----------- |
+| `KEPPEL_TRIVY_ADDITIONAL_PULLABLE_REPOS` | *(optional)* | It adds additional scopes to the token issued by the API and the janitor which is meant to allow the trivy components to pull their DB OCI images from the respective repos. |
+| `KEPPEL_TRIVY_DB_MIRROR_PREFIX` | *(required)* | Prefix under which trivy can find it's database. This might be a mirror or ghcr.io. |
```

Comment:
```suggestion
| `KEPPEL_TRIVY_DB_MIRROR_PREFIX` | *(required)* | Prefix under which trivy can find its database. This might be a mirror or ghcr.io. |
```

Rule of thumb: Put an apostrophe if it can be substituted with "it is".